src/api/app/controllers/webui/image_templates/interconnects_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Webui
+  module ImageTemplates
+    class InterconnectsController < Webui::WebuiController
+      def index
+        @projects = Project.image_templates
+        respond_to do |format|
+          format.xml
```

Comment:
Well, we talked about it in our meeting yesterday as you know. IIRC no one disagreed to start going into that direction. Though we didn't come out with a real commitment.